src/Pie.js

```diff
@@ -77,15 +76,15 @@ const RoundDividers = ({ paintedSections, dividerSize, width, radius, background
 };
 
 const getArcAngle = (percentage) => percentage / 100 * 360;
-const shouldShowDivider = (percentage, dividerSize) => percentage <= 100 && !Number.isNaN(dividerSize);
+const shouldShowDivider = (sections) => sections.length > 1;
 
 const Pie = ({ sections, radius, innerRadius, backgroundColor, strokeCap, dividerSize }) => {
   const width = radius - innerRadius;
   const backgroundPath = createPath(radius, radius, radius - width / 2, 0, 360);
   const shouldShowRoundDividers = !!dividerSize && strokeCap === 'round';
   let startValue = 0;
   let paintedSections = [];
-
+  const shouldShowDividerForSection = shouldShowDivider(sections);
```

Comment:
Can rename to `showDividers` or something because this isn't per section anymore